doc/manuals/user/known_limitations.md

```diff
@@ -29,18 +29,30 @@ Or, if you are sending a huge request, this one:
 carefully" text. Developers of the HTTP library in which Orion Context
 Broker is based seem to be funny guys :) :)
 
-If you find this 1MB limit too coarse, send us an email so we can
-consider your feedback in future releases.
+If this 1 MB limit doesn't work for you, you can change it using the CLI option -payloadMaxSize.
+But, this is untested territory. At your own risk ...
 
 ## Notification maximum size
 
-Notification maximum size is set to 8MB. Larger notifications will not be sent by context broker and you
+The maximum size of notification payloads defaults to 8MB. Larger notifications will not be sent by the context broker and you
 will get the following trace in the log file:
 
     HTTP request to send is too large: N bytes
 
 where N is the number of bytes of the too large notification.
 
+However, you can change this limit by starting the broker with the CLI option `-notifMaxSize <size in bytes>`
+
+Note also that there is another message size limit, one that is valid for **all** outgoing messages from the broker:
+
+* Normal responses
+* Notifications
+* Forwarded messages
+
+This last message size limit is about the size of the entire message, not only the payload and its default value is 8MB, which can
+be modified using the CLI option `-outMsgMaxSize`.
+
+
```

Comment:
I plan to do a PR on this one to fix documentation. Thus, you can ignore changes in documentation if you want.

Reply:
I happily leave this to you :)

Reply:
Fixed by f46c34a3c403eed0ac8f65a86b49a47d6d46942b